assembl/static2/js/app/pages/landingPageAdmin.jsx

```diff
@@ -68,33 +101,72 @@ class LandingPageAdmin extends React.Component<Props, State> {
           displayAlert('danger', error, false, 30000);
         });
     }
+
+    if (pageHasChanged) {
+      updateDiscussion({
+        variables: {
+          titleEntries: page.titleEntries,
+          subtitleEntries: convertEntriesToHTML(page.subtitleEntries),
+          buttonLabelEntries: page.buttonLabelEntries,
+          headerImage: this.getImageVariable(page.headerImage),
+          logoImage: this.getImageVariable(page.logoImage)
+        }
+      })
+        .then(() => {
+          displayAlert('success', I18n.t('administration.landingPage.successSave'));
+        })
+        .catch((error) => {
+          displayAlert('danger', error.message);
+        });
+    }
   };
 
+  dataHaveChanged = (): boolean => this.props.landingPageModulesHasChanged || this.props.pageHasChanged;
+
   render() {
-    const { landingPageModulesHasChanged, section } = this.props;
+    const { section } = this.props;
     const currentStep = parseInt(section, 10);
-    const saveDisabled = !landingPageModulesHasChanged;
+    const saveDisabled = !this.dataHaveChanged();
     return (
       <div className="landing-page-admin">
         <SaveButton disabled={saveDisabled} saveAction={this.saveAction} />
         {section === '1' && <ManageModules {...this.props} />}
-        {!isNaN(currentStep) && <Navbar currentStep={currentStep} totalSteps={1} phaseIdentifier="landingPage" />}
+        {section === '2' && <CustomizeHeader {...this.props} />}
+        {!isNaN(currentStep) && <Navbar currentStep={currentStep} totalSteps={2} phaseIdentifier="landingPage" />}
       </div>
     );
   }
 }
 
-const mapStateToProps = ({ admin: { landingPage } }) => ({
-  landingPageModulesHasChanged: landingPage.modulesHasChanged,
-  landingPageModules: landingPage.modulesByIdentifier
-    .map((module) => {
-      const identifier = module.getIn(['moduleType', 'identifier']);
-      const idx = landingPage.enabledModulesInOrder.indexOf(identifier);
-      return module.set('order', idx + 1).set('_isNew', !module.get('existsInDatabase'));
-    })
-    .valueSeq()
-    .toJS()
-});
+const mapStateToProps = ({ admin: { editLocale, landingPage } }) => {
+  const { page } = landingPage;
+  const subtitle = getEntryValueForLocale(page.get('subtitleEntries'), editLocale, '');
+  return {
+    landingPageModulesHasChanged: landingPage.modulesHasChanged,
+    landingPageModules: landingPage.modulesByIdentifier
+      .map((module) => {
+        const identifier = module.getIn(['moduleType', 'identifier']);
+        const idx = landingPage.enabledModulesInOrder.indexOf(identifier);
+        return module.set('order', idx + 1).set('_isNew', !module.get('existsInDatabase'));
+      })
+      .valueSeq()
+      .toJS(),
+    header: {
+      title: getEntryValueForLocale(page.get('titleEntries'), editLocale, ''),
```

Comment:
when we will add all other modules admin, I think it will be necessary to put this stuff in a function to avoid a long and difficult to maintain `mapStateToProps`